app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -37,7 +37,7 @@ public function store(Request $request)
         $request->validate([
             'user_first_name' => ['required', 'string', 'min:2', 'max:255'],
             'user_last_name' => ['required', 'string', 'min:2', 'max:255'],
-            'user_email' => ['required', 'string', 'email', 'unique:users', 'max:255', 'unique:users'],
+            'user_email' => ['required', 'string', 'email', 'max:255', 'unique:users'],
```

Comment:
tại sao nh` tk có thể có chung 1 email?

Reply:
dạ cái này do xung e resolve nhằm merge conflict ạ, e hơi ẩu chỗ này ạ

Reply:
mà e mới test lại a. nếu dky tài khoản mà đã có email đã sử dụng rồi thì nó không cho phép đăng ký nữa, và nó sẽ hiển thị lỗi The user email has already been taken

Reply:
e xóa bớt 1 cái rule á a, tại nó bị trùng á